features/admin.connections.v1/components/authenticator-grid.tsx

```diff
@@ -346,10 +385,55 @@ export const AuthenticatorGrid: FunctionComponent<AuthenticatorGridPropsInterfac
      */
     const handleGridItemOnClick = (e: SyntheticEvent, authenticator: ConnectionInterface
         | AuthenticatorInterface): void => {
-        handleAuthenticatorEdit(authenticator.id);
+        handleAuthenticatorEdit(authenticator.id, authenticator.type);
         onItemClick && onItemClick(e, authenticator);
     };
 
+    /**
+     * Checks if the delete button should be enabled.
+     *
+     * @param authenticator - Authenticator.
+     * @returns - True if the delete button is enabled, false otherwise.
+     */
+    const isDeleteEnabled = (authenticator: ConnectionInterface): boolean => {
+        if (authenticator.type === ConnectionTypes.IDVP) {
+            return hasIdVPDeletePermission;
+        }
+
+        if (!hasConnectionDeletePermission) {
+            return false;
+        }
+
+        return ConnectionsManagementUtils.isConnectorIdentityProvider(authenticator)
+            || (authenticator as ConnectionInterface).type === AuthenticatorTypes.FEDERATED;
+    };
+
+    const resolveResourceImage = (
```

Comment:
Let's add the doc comments for this method

Reply:
Fixed via d89d804b4490c34fa96959d8550072fd7df05403